lib/vanagon/driver.rb

```diff
@@ -11,7 +11,14 @@ class Vanagon
   class Driver
     include Vanagon::Utilities
     attr_accessor :platform, :project, :target, :workdir, :remote_workdir, :verbose, :preserve
-    attr_accessor :timeout, :retry_count
+
+    def timeout()
+      @timeout ||= @project.timeout || ENV["VANAGON_TIMEOUT"] || 7200
+    end
+
+    def retry_count()
```

Comment:
[nitpick] fairly consistently throughout the codebase when we have a def that takes no argument we use `def name` instead of `def name()`